acinclude/os-deps.m4

```diff
@@ -800,6 +800,8 @@ AC_DEFUN([SQUID_CHECK_FUNCTIONAL_CPU_PROFILER],[
 // okay
 #elif defined(_M_IX86) && defined(_MSC_VER) /* x86 platform on Microsoft C Compiler ONLY */
 // okay
+#elif defined(HAVE_CLOCK_GETTIME_NSEC_NP) && defined(CLOCK_MONOTONIC_RAW)
```

Comment:
In my tests, this check fails without `#include <ctime>` on line 795 (i.e. the first line of this `AC_LANG_SOURCE` snippet) because `CLOCK_MONOTONIC_RAW` is not defined without that header in my environment. I warned about the lack of that `#include` [earlier](https://github.com/squid-cache/squid/pull/757#discussion_r557811814), but my warning was probably lost in the noise. Please fix.

Reply:
ok did the changes and tested old autoconf settings a bit messed up in my side made me missed this. good point.

Reply:
next time, I ll probably just file a bug report, that will probably be a lot faster, I did not envision this like this.

Reply:
I feel your pain! FWIW, if you just filed a bug report, it would have probably stayed as such for a while because nobody would volunteer to convert it into a pull request. Most pull requests do not require so many rewrites (or those rewrites are done by seasoned Squid developers). This PR is somewhat exceptional because it deals with neglected low-level code on a platform most of us do not have access to.